ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/UserManagementServiceIT.scala

```diff
@@ -360,15 +356,9 @@ final class UserManagementServiceIT extends LedgerTestSuite {
       assert(res2.nextPageToken.nonEmpty, s"Second next page token should be non-empty")
       assertLength("second page", 3, res2.users)
 
-      assert(res3.nextPageToken.nonEmpty, s"Third next page token should be non-empty")
+      assert(res3.nextPageToken.isEmpty, s"Third next page token should be non-empty")
```

Comment:
```suggestion
      assert(res3.nextPageToken.isEmpty, "Third next page token should be empty")
```
There are also some other unnecessary interpolations if you want to remove them.

Reply:
Thanks, will check interpolations in another PR as I'm revisiting those tests also from yet another angle